components/kyma-environment-broker/internal/component_input_helpers.go

```diff
@@ -76,67 +66,6 @@ func getBTPOperatorProvisioningOverrides(creds *ServiceManagerOperatorCredential
 	}
 }
 
-func GetBTPOperatorReconcilerOverrides(creds *ServiceManagerOperatorCredentials, clusterIdGetter ClusterIDGetter) ([]reconcilerApi.Configuration, error) {
```

Comment:
I had to move this from `components/kyma-environment-broker/internal` to `components/kyma-environment-broker/internal/storage` due to import cycle
```
env CGO_ENABLED=0 go build -o kyma-environment-broker ./cmd/broker/main.go
package command-line-arguments
        imports github.com/kyma-project/control-plane/components/kyma-environment-broker/common/hyperscaler
        imports github.com/kyma-project/control-plane/components/kyma-environment-broker/internal
        imports github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage
        imports github.com/kyma-project/control-plane/components/kyma-environment-broker/internal: import cycle not allowed
make: *** [/home/wozy/projects/go/src/github.com/kyma-project/control-plane/scripts/generic_make_go.mk:128: build-local] Error 1
```
I realize this is not ideal but I don't think I can figure out a better path forward in timely manner. Well keep this in mind for future refactor